### DISCUSSION TITLE
Pencil electrode test benchmark for corrosion simulations

### DISCUSSION THREAD

**Discussion by arjunvarmar** [idaholab/moose#27190] [Q&A Modules: Phase field]
I am trying to run a simple Kim-Kim-Suzuki model that can be used to obtain the results of the pencil electrode test (see [1](https://iopscience.iop.org/article/10.1149/1945-7111/ac5349) and [2](https://doi.org/10.1016/j.corsci.2016.04.001)). I use the same input script as given in the examples of the phase field module in MOOSE and only modify the initial and boundary conditions to suit the benchmark problem. However, when I run the simulation, the interface is stationary even though it becomes diffuse and smooth in the first few steps. I am using non-dimensionalised values of parameters in the script. I am attaching my script (.i file) for reference. (The extension has been changed to .txt as .i is not accepted for upload.)

It would be a great help if someone who has already solved this benchmark could let me know what mistake I am making here.

[corr_trial.txt](https://github.com/idaholab/moose/files/14750211/corr_trial.txt)

**Comment by zyyc1203-ops** [comment]
bro，have u got the answer？i got a similar question，needing your help，sincerely
